tests/unit_tests/key_container.cpp

```diff
@@ -0,0 +1,41 @@
+#include <boost/filesystem/path.hpp>
+#include <cstdint>
+#include <gtest/gtest.h>
+
+#include "crypto/chacha.h"
+#include "seraphis_wallet/key_container.h"
+#include "unit_tests_utils.h"
+
+using namespace seraphis_wallet;
+
+TEST(seraphis_wallet, key_container) {
+    KeyContainer container;
+    crypto::chacha_key key;
+}
```

Comment:
This looks incomplete!?

Reply:
wasn't removed from the old commit, will remove